src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -98,7 +98,7 @@ class EstimateYourBenefitsForm extends React.Component {
     }
   };
 
-  handleCalculateBenefitsClick = childSection => {
+  handleCalculateBenefitsClick = async (accordionButtonId, childSection) => {
```

Comment:
```suggestion
  handleCalculateBenefitsClick = async childSection => {
  const accordionId = `${createId(childSection)}-accordion`;
```

instead of doing `${createId(childSection)}-accordion` in 4 places this can be done in this method

Reply:
Done